composer.json

```diff
@@ -2,40 +2,46 @@
     "name": "olcs/olcs-common",
     "description": "Common library for the OLCS Project",
     "type": "library",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "type": "composer",
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "only": [
-                "olcs/olcs-logging",
-                "olcs/olcs-utils"
-            ]
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
-        "php": ">=7.4",
-        "olcs/olcs-laminas": "~2.5.0 || ~3.0.0"
+        "php": "^7.4",
+        "laminas/laminas-authentication": "^2.6",
+        "laminas/laminas-form": "^2.13",
+        "laminas/laminas-servicemanager": "^2.7",
+        "laminas/laminas-cache": "^2.8",
+        "laminas/laminas-eventmanager": "^2.6",
+        "laminas/laminas-http": "^2.8",
+        "laminas/laminas-i18n": "^2.14",
+        "laminas/laminas-modulemanager": "^2.9",
+        "laminas/laminas-mvc": "^2.7",
+        "laminas/laminas-view": "^2.11",
+        "laminas/laminas-validator": "^2.11",
+        "olcs/olcs-utils": "^5.0",
+        "dvsa/authentication-cognito": "^2.0",
+        "laminas/laminas-console": "^2.7",
+        "laminas/laminas-stdlib": "^2.7",
+        "laminas/laminas-zendframework-bridge": "^1.6",
+        "container-interop/container-interop": "^1.2",
+        "laminas/laminas-psr7bridge": "^0.2.2"
     },
     "require-dev": {
+        "bamarni/composer-bin-plugin": "^1.8",
+        "dms/phpunit-arraysubset-asserts": "^0.5.0",
         "doctrine/annotations": "^1.14.2",
-        "olcs/olcs-devtools": "~4.206.0",
+        "ezyang/htmlpurifier": "^4.17",
+        "johnkary/phpunit-speedtrap": "^4.0",
+        "laminas/laminas-code": "^3.4.1",
+        "laminas/laminas-navigation": "2.13.0",
+        "laminas/laminas-serializer": "^2.5",
+        "lm-commons/lmc-rbac-mvc": "^2.6",
+        "mikey179/vfsstream": "~v1.6.11",
+        "mockery/mockery": "^1.6.7",
         "olcs/olcs-logging": "^5.0.0",
-        "olcs/olcs-transfer": "~4.206.0",
-        "olcs/olcs-utils": "^5.0.0",
-        "phpstan/phpstan": "^0.12.100",
+        "olcs/olcs-transfer": "^5.0.0",
+        "phpunit/phpunit": "^9.6",
         "soflomo/purifier": "^1.0",
-        "symfony/lock": "^5"
+        "symfony/filesystem": "^5.4",
+        "symfony/lock": "^5",
+        "laminas/laminas-inputfilter": "^2.10"
```

Comment:
Used in source code.

Reply:
filesystem, symfony lock  has been used in test files

Reply:
`laminas/laminas-inputfilter` is used in the source code. Could we move it to `dependencies` please?

Reply:
moved